ovh/data_iam_policies.go

```diff
@@ -0,0 +1,46 @@
+package ovh
+
+import (
+	"context"
+	"sort"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/ovh/terraform-provider-ovh/ovh/helpers/hashcode"
+)
+
+func dataSourceIamPolicies() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
```

Comment:
the endpoint actually returns policies and their body, but I man only keeping the id. Should I change it so this also contains policies content ?